client/src/containers/Dataset.jsx

```diff
@@ -96,6 +98,15 @@ class Dataset extends Component {
     }));
   }
 
+  handleNavigateToVisualise({ datasetId }) {
```

Comment:
Perhaps this method could just grab the dataset ID from props? It might be a bit more direct, and probably less code than looking up the params in the child component and then passing the id back up. `getId(this.props.dataset)` would work I think

Reply:
youre right @gabemart Ill change it

Reply:
k done